Makefile

```diff
@@ -567,11 +568,13 @@ openstack_prep: export IMAGE=${OPENSTACK_IMG}
 openstack_prep: $(if $(findstring true,$(BMO_SETUP)), crc_bmo_setup) ## creates the files to install the operator using olm
 openstack_prep: $(if $(findstring true,$(NETWORK_BGP)), nmstate nncp netattach metallb metallb_config)
 openstack_prep: $(if $(findstring true,$(NETWORK_ISOLATION)), nmstate nncp netattach metallb metallb_config)
+openstack_prep: $(if $(findstring true,$(INSTALL_CERT_MANAGER)), certmanager)
```

Comment:
if the ordering matters here, it should go first, since certmanager was before openstack_prep.

Reply:
Yes and it worked.!